src/KingSystem/Event/evtInfoData.cpp

```diff
@@ -0,0 +1,72 @@
+#include "KingSystem/Event/evtInfoData.h"
+#include "KingSystem/Resource/resLoadRequest.h"
+#include "KingSystem/Utils/Byaml/Byaml.h"
+#include "basis/seadNew.h"
```

Comment:
```suggestion
#include <basis/seadNew.h>
```

(sead should be treated like an external library)